lqr_node: 
  ros__parameters:
    k1_gain: 1.0
    k2_gain: 1.0
    k3_gain: 1.0
    k4_gain: 1.0
    k1_coeff: [10.0, 10.0, 10.0]
    k2_coeff: [1.0, 1.0, 1.0]
    k3_coeff: [10.0, 10.0, 10.0]
    k4_coeff: [1.0, 1.0, 1.0]
    zero_speed  : 0.0
    max_speed : 0.35 
    min_speed : 0.2 
    error_threshold : 0.16 
    max_right_steering : 0.4 
    max_left_steering : -0.4
    Ts : 0.01
